Library/Tracking/KSRAnalytics.swift

```diff
@@ -1149,9 +1148,23 @@ public final class KSRAnalytics {
 
   // MARK: - Search Events
 
-  /// Call once when the search view is initially shown.
-  public func trackProjectSearchView() {
-    self.track(event: ApprovedEvent.searchPageViewed.rawValue, page: .search)
+  /// Call whenever the search view is shown.
+  public func trackProjectSearchView(params: DiscoveryParams, results: Int) {
+    var props = discoveryProperties(from: params)
+
+    if let query = params.query {
+      props = props.withAllValuesFrom(["discover_search_term": query])
+    }
+
+    if results > 0 {
```

Comment:
I don't think this conditional is necessary? If it's 0 that is probably meaningful information for insights.